app/jobs/submit_patron_request_job.rb

```diff
@@ -21,4 +20,32 @@ def perform(patron_request) # rubocop:disable Metrics/AbcSize
 
     patron_request.update(illiad_response_data:, folio_responses:)
   end
+
+  private
+
+  # Conditions under which an item should be requested via ILLiad
+  # The request is a scan, OR
+  #   The patron is ILB eligible, AND
+  #   The request is a recall, AND
+  #   The item can be recalled, AND
+  #     The item has a status indicating it won't be available soon, OR
+  #     The item has an existing request queue (TODO: https://github.com/sul-dlss/sul-requests/issues/2234)
+  def send_to_illiad?(patron_request, item)
+    return true if patron_request.scan?
+    return false unless patron_request.patron.ilb_eligible?
+
+    patron_request.fulfillment_type == 'recall' &&
+      item.hold_recallable?(patron_request.patron) &&
+      !item.status.in?(folio_recall_statuses)
```

Comment:
What do you think about moving the item status piece up to a guard clause too? 


```suggestion
    return false unless patron_request.patron.ilb_eligible?
    return false if item.status.in?(folio_recall_statuses)

    patron_request.fulfillment_type == 'recall' &&
      item.hold_recallable?(patron_request.patron)
```

Reply:
for sure!